tests/functional/z_cluster/cluster_expansion/test_resize_osd.py

```diff
@@ -296,3 +299,20 @@ def test_resize_osd_for_large_diff(self, size_to_increase):
         self.prepare_data_before_resize_osd()
         resize_osd(self.new_storage_size)
         self.verification_steps_post_resize_osd()
+
+    @tier1
+    @tier4
```

Comment:
Is there a need to run this test on multiple tiers? having it covered in a single tier execution is not sufficient

Reply:
Yes, correct. I fixed it.